setup.py

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env python
 
-from Cython.Distutils import build_ext
-from setuptools import setup, Command, Extension
+from setuptools import dist, setup, Command, Extension
 import os.path
 
+# Install Cython before building package.
+dist.Distribution().fetch_build_eggs(['cython>=0.28.5'])
+
+from Cython.Distutils import build_ext
```

Comment:
If I follow, this will needlessly download cython even if it's already installed (e.g. as a distro package).

Given that not doing this just requires the user run `pip install cython` as a one-time thing first, I don't think this change is a good one.

Also, the Cython requirement appears to already be specified in `setup.py`:

```
      setup_requires=['Cython>=0.28.5,<1.0'],                      
```

If that doesn't work, fixing it seems a better thing to do.